docs/setting-up/client/postgresql.md

```diff
@@ -77,7 +77,7 @@ Decide which database extension to use, and configure your database server for i
 We recommend choosing only one of these. **If you use both, you will get duplicate metrics.**
 
 !!! caution alert alert-warning "Caution"
-    While we recommend use of the newer `pg_stat_monitor` extension, be aware it is currently in beta phase and unsupported.
+    The current version of  `pg_stat_monitor` 1.0.0-rc2 does not work with PostgreSQL versions 11 and 12 and will be fixed in the subsequent releases.
```

Comment:
I don't think this is an accurate description. PGSM does work with these PG versions, it's the **integration with PMM** that is currently affected (and a fix for this is in the works here: https://github.com/percona/pg_stat_monitor/pull/201

Reply:
lets rephrase it as: there is know issue with ... link to the ticket. Still leave that is not fully supported yet.